packages/desktop-client/src/components/manager/BudgetList.tsx

```
@@ -126,66 +138,140 @@
       break;
     case 'local':
+      Icon = SvgFileDouble;
+      status = 'Local';
+      break;
     case 'broken':
+      ownerName = 'unknown';
+      Icon = SvgFileDouble;
+      status = t('Local');
+      break;
+
+    case 'broken':
```

Biome

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)